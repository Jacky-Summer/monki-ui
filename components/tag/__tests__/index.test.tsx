import React from 'react'
import { mount } from 'enzyme'
import Tag from '../tag'

describe('Tag', () => {
  it('render normal Tag correctly', () => {
    const wrapper = mount(<Tag text="Tag1">normal</Tag>)

    expect(wrapper).toMatchSnapshot()
  })
})
