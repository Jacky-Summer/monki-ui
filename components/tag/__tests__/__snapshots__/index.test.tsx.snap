// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tag render normal Tag correctly 1`] = `
<Tag
  closable={false}
  color="primary"
  text="Tag1"
>
  <Transition
    animation="zoom-in-left"
    appear={true}
    in={true}
    timeout={300}
    unmountOnExit={true}
    wrapper={false}
  >
    <CSSTransition
      appear={true}
      classNames="zoom-in-left"
      in={true}
      timeout={300}
      unmountOnExit={true}
    >
      <Transition
        appear={true}
        enter={true}
        exit={true}
        in={true}
        mountOnEnter={false}
        onEnter={[Function]}
        onEntered={[Function]}
        onEntering={[Function]}
        onExit={[Function]}
        onExited={[Function]}
        onExiting={[Function]}
        timeout={300}
        unmountOnExit={true}
      >
        <div
          className="mk-tag tag-primary"
        >
          <span
            className="tag-text"
          >
            Tag1
          </span>
        </div>
      </Transition>
    </CSSTransition>
  </Transition>
</Tag>
`;
